GOVERNANCE.md

```diff
@@ -2,94 +2,174 @@
 
 # Governance
 
-This document outlines the governance model for Astro. This includes the contributor model, code review, merging, and the consequences and process for Code of Conduct violations.
+This document outlines the governance model for Astro. This includes the contributor model, code review process, PR merge process, and the consequences of Code of Conduct violations.
 
-**All members must follow the [Code of Conduct](CODE_OF_CONDUCT.md).** Consequences for member violations are detailed in [Moderation](#moderation).
+👉 **All community members must follow the [Code of Conduct (CoC)](CODE_OF_CONDUCT.md).**  
+Consequences for CoC violations are detailed in [Moderation](#moderation).
+
+👉 **Want to trigger a vote, nomination, or perform some other action?**  
+Scroll down to [Playbook](#playbook).
 
 ## Get Involved
 
-Anyone can become an Astro Contributor regardless of skill level, experience, or background. All types of contribution are meaningful. Our membership system was designed to reflect this.
+**Anything that supports the Astro community is considered a contribution.** All types of contribution are meaningful, from code to documentation to blog posts. Anyone can become an Astro Contributor (yes, even you!). Our goal is to recognize all contributors to Astro regardless of skill level, experience or background.
+
+**@FredKSchott** wrote up some notes on the personal value of open source, which you can read in our [CONTRIBUTING.md](CONTRIBUTING.md#why-contribute-to-open-source) document.
```

Comment:
That's a nice anecdote but it feels a little out of place in the contributing doc which is otherwise instructional. Would it be better to inline that anecdote here (where it seems more appropriate imo) or link to it being posted somewhere else?

I read the contributing doc as "How to contribute" and I read this doc as "These are our values and how we govern them" which is why it makes sense to me here and not there.

Reply:
I'm not sure if it makes any more sense in "GOVERNANCE", but I do see what you mean about CONTRIBUTING being more of a manual. 

Does CONTRIBUTING need to be purely instructional? Is there a specific problem with the first section being a bit more of a "why" / "so what" to a potentially interested contributor?

Reply:
Not so much that it's a "why" section, but that it's a personal anecdote. Another contributor couldn't exactly submit a PR that removes one of those bullet points, it's your anecdote not there's. That makes it feel like less of a collaborative document. If it were instead linked to from a blog post or something like that it would not feel that way to me.

Reply:
+1, that's a great point. Let me move to CONTRIBUTING and see if it works there. I could also post it somewhere more personal, like a personal blog and then just link there.

Reply:
+1, that's a great point. ~~Let me move to CONTRIBUTING and see if it works there. I could also post it somewhere more personal, like a personal blog and then just link there.~~ Err sorry, got confused about where this was. 

Okay, I'm +1 for moving to GOVERNANCE for now, and then a personal blog later once I get mine back into good shape.

Reply:
err, ya know what, I don't like this as much as when I wrote it :) I'll just remove this for now, and write it up on a personal blog later. Thanks for the sanity check on this!

Reply:
Even a gist would be fine and linking, we could potentially link to many things in our document.